{
  new AssertThrows(IllegalArgumentException.class){
    public void test() throws Exception {
      TestBean tb=new TestBean();
      Errors errors=new BeanPropertyBindingResult(tb,"tb");
      ValidationUtils.invokeValidator(null,tb,errors);
    }
  }
.runTest();
}
